### PR TITLE
Merge upstream op-geth v1.101304.1

### PR DESCRIPTION
**Pull request by bnoieh**
### Description

Merge upstream op-geth [v1.101304.1](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101304.1)

### Rationale

To import bugsfix/improvements/features from upstream op-geth

### Example

n/a
#### changes from geth
[v1.11.6](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.6) -> [v1.13.4](https://github.com/ethereum/go-ethereum/releases/tag/v1.13.4)
#### changes from op-geth
[v1.101105.2](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101105.2) ➝ [v1.101304.1](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101304.1)

here's modification in this pr for changes from op-geth:
1. disable super-chain
2. disable Deploys the create2Deployer to [0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2](https://github.com/ethereum-optimism/specs/blob/main/specs/predeploys.md#create2deployer)
3. use our own eip-1559 parameter for `EIP1559DenominatorCanyon` 
#### changes from bnb-chain/op-geth develop branch
1. revert [perf: concurrency and memory improvements for execution layer](https://github.com/bnb-chain/op-geth/commit/f80e72bcd2b00f738326e37c96ae150dbc9fa4d4) 
2. revert https://github.com/bnb-chain/op-geth/pull/52 due to incompatible with upstream
3. some refactor and code clean for previous code

### CODE REVIEW

**Comment by welkin22** [comment]
TL;DR
Due to the large size of the PR, I only carefully checked the parts I have modified before, and only briefly looked over the rest. Currently, I haven't found any problems.
